### ISSUE TITLE
SystemSizing object N2, \field Central Heating Maximum System Air Flow Ratio

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/OpenStudio#2718]
EnergyPlus idd has "Central Heating Maximum System Air Flow Ratio" as N2, while OpenStudio idd has "Minimum System Air Flow Ratio" as "N2"

OpenStudio has as system_sizing.setMinimumSystemAirFlowRatio, but not setMinimumSystemAirFlowRatio

I didn't look in the translator code, but below is screenshot of IDD, and from testing the value in the OSM for "Minimum System Air Flow Ratio" is saved in forward translated IDF as "Central Heating Maximum System Air Flow Ratio".

Also checked that "Central Heating Maximum System Air Flow Ratio" doesn't show up in the OpenStudio idd while "Minimum System Air Flow Ratio" doesn't show up in the EnergyPlus idd.

![screen shot 2017-08-09 at 2 14 17 pm](https://user-images.githubusercontent.com/4857055/29142056-e01aa8e8-7d0d-11e7-9dc6-51e66cf033a1.png)

**Comment by DavidGoldwasser** [comment]
Does appear to be used in a number of measures
https://github.com/NREL/OpenStudio-measures/search?utf8=%E2%9C%93&q=setMinimumSystemAirFlowRatio&type=

**Comment by MatthewSteen** [comment]
I noticed this recently too. Here's the FT code:

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateSizingSystem.cpp#L84

Looks like the field was changed in EP between [version 8.2](http://bigladdersoftware.com/epx/docs/8-2/input-output-reference/group-design-objects.html#field-minimum-system-air-flow-ratio) and [version 8.3](http://bigladdersoftware.com/epx/docs/8-3/input-output-reference/group-design-objects.html#field-minimum-system-air-flow-ratio), but they are used in the same way.

>Field: Minimum System Air Flow Ratio
The design minimum central air flow ratio. This ratio is the minimum system air flow rate divided by the maximum system air flow rate. The value must be between 0 and 1. For constant volume systems the ratio should be set to 1. Note that this ratio should be set to reflect what the user expects the system flow rate to be when maximum heating demand occurs. This ratio is used in calculating the central system heating capacity. Thus if the system is VAV with the zone VAV dampers held at minimum flow when there is a zone heating demand, this ratio should be set to the minimum flow ratio. If the zone VAV dampers are reverse action and can open to full flow to meet heating demand, this ratio should be set to 1.

>Field: Central Heating Maximum System Air Flow Ratio
The ratio of the maximum system air flow rate for heating to the maximum system air flow rate. 
The value must be between 0 and 1. For constant volume systems the ratio should be set to 1. This ratio should be set to reflect what the user expects the system flow rate to be when maximum heating demand occurs. This ratio is used in calculating the central system heating capacity. Thus if the system is VAV with the zone VAV dampers held at minimum flow when there is a zone heating demand, this ratio should be set to the minimum flow ratio. If the zone VAV dampers are reverse action and can open to full flow to meet heating demand, this ratio should be set to 1. The default is set to 0.5, reflecting the fact that VAV dampers are typically not allowed to fully open during heating.
